src/Illuminate/Foundation/Configuration/ApplicationBuilder.php

```diff
@@ -300,6 +300,8 @@ protected function withCommandRouting(array $paths)
         $this->app->afterResolving(ConsoleKernel::class, function ($kernel) use ($paths) {
             $this->app->booted(fn () => $kernel->addCommandRoutePaths($paths));
         });
+
+        return $this;
```

Comment:
It is indeed missing, but isn't the method `withCommandRouting()`? 

Instead of `withBindings()` as in your description?

Reply:
It's withCommandRouting. Thank you!